### PR TITLE
Update Calling SDK beta to 1.13.2-beta.1

### PR DESCRIPTION
**Pull request by JamesBurnside**
# What
Update Calling SDK beta to 1.13.2-beta.1

# Why
Housekeeping

# How Tested
n/a - Ci should test this

### CODE REVIEW

**Review by dmceachernmsft — APPROVED** [review]

File: packages/calling-stateful-client/src/LocalVideoStreamVideoEffectsSubscriber.ts [hunk 1]
```diff
@@ -48,17 +48,17 @@ export class LocalVideoStreamVideoEffectsSubscriber {
     this._localVideoStreamEffectsAPI.off('effectsError', this.effectsError);
   };
 
-  private effectsStarted = (effectName: VideoEffectName): void => {
+  private effectsStarted = (effects: VideoEffectName[]): void => {
     this.updateEffectsState({
       isActive: true,
-      effectName: effectName
+      effectName: effects[0]
```
Comment (JamesBurnside) [inline packages/calling-stateful-client/src/LocalVideoStreamVideoEffectsSubscriber.ts:54]: As part of background effects we are updating the stateful to also provide an array of effects, this will be a separate PR

**Review by prabhjot-msft — APPROVED** [review]
